### PR TITLE
[13.0] s_product_*_multi_link: fix non symmetric link

### PR DESCRIPTION
**Pull request by simahawk**
and split mixin to its own file

### CODE REVIEW

**Comment by simahawk** [comment]
@chafique-delli if multi links modules are on your radar for v14 you might want to review this and get it merged before you start w/ them. FTR: already used in prod since a while.

File: shopinvader_product_template_multi_link/tests/test_product.py [hunk 1]
```diff
@@ -54,7 +54,9 @@ def setUpClass(cls):
         cls.shopinvader_variant_3_2 = cls.variant_3_2._get_invader_variant(
             cls.backend, "en_US"
         )
-
+        cls.link_type_sym = cls.env["product.template.link.type"].create(
```
Comment (chafique-delli) [inline shopinvader_product_template_multi_link/tests/test_product.py]: Replace 'cls.link_type_sym' by 'cls.link_type_asym' to be consistent with 'is_symmetric: False'

**Review by chafique-delli — APPROVED** [review]
Code Review.
It's ok for me.

File: shopinvader_product_template_multi_link/tests/test_product.py (hunk 1 above)
Comment (simahawk) [reply shopinvader_product_template_multi_link/tests/test_product.py]: done

**Pull request merged**